src/cheshire/custom.clj

```diff
@@ -35,8 +35,9 @@
                         ^JsonFactory (or *json-factory* json-factory) sw)]
          (when (:pretty opt-map)
            (.useDefaultPrettyPrinter generator))
-         (when (:escape-non-ascii opt-map)
-           (.enable generator (.mappedFeature JsonWriteFeature/ESCAPE_NON_ASCII)))
+         (when (some? (:escape-non-ascii opt-map))
```

Comment:
Adding `(some? …)` here means that we don't handle the value `false` correctly, since `(some? false)` returns true.

Reply:
Thanks for taking a peek!

This is deliberate (hopefully deliberately correct!). The factory opts can specify `:escape-non-ascii` of `true` and the fn opts might want to override that with `false`. Notice also that I switched from `.enable` to `.configure` for the fn opts.

So now we only want to ignore the `:escape-non-ascii` fn opt when it is `nil`.

Does that make sense?

Reply:
Ahh yep, I failed to notice that you switched from `.enable` to `.configure`. I was wondering why it still worked locally but appeared incorrect. Thanks for the clarification!